webapp/home/templates/home/snippets/about/about.html

```diff
@@ -42,8 +41,10 @@ <h3 id="operational-partners">
   The service is underpinned by computational resources provided by
   <a href="https://www.unimelb.edu.au/" target="_blank">The University of Melbourne</a>,
   <a href="https://www.qcif.edu.au/" target="_blank">QCIF</a>,
-  <a href="https://pawsey.org.au/" target="_blank">Pawsey Supercomputing Centre</a>,
+  <a href="https://pawsey.org.au/" target="_blank">Pawsey Supercomputing Research Centre</a>,
+  <a href="https://nci.org.au/" target="_blank">National Computational Infrastructure</a>
   <a href="https://www.aarnet.edu.au/" target="_blank">AARNet</a>
```

Comment:
These two lines need a comma at the end (see line 43)

Reply:
Added, thanks!